### PR TITLE
[SPARKTA-256] [PLUGINS] Kafka data Formats

### PR DESCRIPTION
**Pull request by compae**
#### Description

The data saved into Kafka is formatted with json or row with separator.

#### Test
All OK

### CODE REVIEW

**Comment by acemilio** [comment]
good job!
+1

**Comment by Gschiavon** [comment]
+1

File: plugins/output-kafka/src/main/scala/com/stratio/sparkta/plugin/output/kafka/KafkaOutputFormat.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+package com.stratio.sparkta.plugin.output.kafka
+
+object KafkaOutputFormat extends Enumeration {
+
+  type Status = Value
+  val ROW, JSON = Value
+
+  def toOutputFormat(format: String) : Value = format.toUpperCase match {
```
Comment (danielcsant) [inline plugins/output-kafka/src/main/scala/com/stratio/sparkta/plugin/output/kafka/KafkaOutputFormat.scala]: Enumeration already implements this function: withName

http://stackoverflow.com/questions/1898932/case-objects-vs-enumerations-in-scala

**Comment by danielcsant** [comment]
Document this new property
https://stratio.atlassian.net/wiki/display/SPARKTA/7.4+Outputs#id-7.4Outputs-8.4.9Kafka

**Comment by compae** [comment]
Changes done!

**Comment by compae** [comment]
Added documentation

**Comment by danielcsant** [comment]
Add changes to template

https://github.com/Stratio/Workflow/blob/master/web/src/main/resources/data-templates/fake_data/items.json

**Pull request merged**